libs/image/CMakeLists.txt

```diff
@@ -33,10 +33,13 @@ target_include_directories(${TARGET} PUBLIC ${PUBLIC_HDR_DIR})
 # ==================================================================================================
 # Compiler flags
 # ==================================================================================================
-target_compile_options(${TARGET} PRIVATE
-        -Wno-deprecated-register
-        $<$<CONFIG:Release>:-ffast-math>
-)
+target_compile_options(${TARGET} PRIVATE -Wno-deprecated-register)
+
+if (MSVC OR CLANG_CL)
+    target_compile_options(${TARGET} PRIVATE $<$<CONFIG:Release>:/fp:fast>)
+else()
+    target_compile_options(${TARGET} PRIVATE $<$<CONFIG:Release>:-ffast-math>)
```

Comment:
These need to be on their own line. It breaks compilation on macOS and Linux:

clang: error: unknown argument: '-ffast-math>'

It applies to all the CMakeLists

Reply:
I found an actual stray > in the CMakeLists for mgen, and fixed that. Could you please see if the build for MacOSX now works again?